terminal.go

```diff
@@ -749,12 +749,12 @@ func (g *LightningTerminal) ValidateMacaroon(ctx context.Context,
 		}
 
 	case isLitURI(fullMethod):
-		wrap := fmt.Errorf("invalid basic auth")
-		_, err := g.rpcProxy.convertBasicAuth(ctx, fullMethod, wrap)
-		if err != nil {
+		if err := g.sessionRpcServer.macaroonService.ValidateMacaroon(
```

Comment:
This should also be guarded by `if !g.sessionRpcServerStarted {` like the other sub servers above.